packages/sdk/src/sdk/utils/mintFixedDecimalMap.ts

```diff
@@ -1,7 +0,0 @@
-import { USDC, wAUDIO, BONK } from '@audius/fixed-decimal'
-
-export const mintFixedDecimalMap = {
```

Comment:
FYI @faridsalau this internal management is gone, you'll have to manage these amounts in the client. Since you'll be using the `decimals` that are returned from the DB, I suggest doing `new FixedDecimal(amount, decimals)` when making the construction